answer_rocket/chat.py

```diff
@@ -150,7 +150,7 @@ def completion(self, model_type: ModelType = 'COMPLETION', stream_callback: Opti
 
         return False, str(last_error)
 
-    def ask_question(self, copilot_id: str, question: str, thread_id: str = None, skip_report_cache: bool = False, dry_run_type: str = None):
+    def ask_question(self, copilot_id: str, question: str, thread_id: str = None, skip_report_cache: bool = False, dry_run_type: str = None, model_overrides: dict = None):
```

Comment:
Should update the docstring to include an explanation for this param and a description of its shape.